### DISCUSSION TITLE
Front panel filament load button?

### DISCUSSION THREAD

**Discussion by Avpman2** [xblax/flashforge_ad5m_klipper_mod#163] [Q&A]
Where on the printer touchscreen, with this mod, is a single button to heat and load filament? Looks like there is more than one step, heat, load, and someway to stop it?

**Comment by consp** [comment]
> with this mod, is a single button to heat and load filament?

There isn't since it depends on the temperature and the type of filament which the printer doesn't know. You set the extruder temperature and then load the filament (and then cool down). So the minimum is two buttons, unleass you already set the temperature a different way.

The original flashforge way is overly simplistic and no open source GUI project does it that way afaik.

Stop is "automatic" in the sense that you only extrude a set amount of material.

**Comment by Avpman2** [comment]
👍
